baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/DuplicateArgumentTypesTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import org.junit.jupiter.api.Test;
+
+public final class DuplicateArgumentTypesTest {
+
+    private static final BugCheckerRefactoringTestHelper.TestMode TEST_MODE =
+            BugCheckerRefactoringTestHelper.TestMode.AST_MATCH;
+
+    @Test
+    void testSameType() {
+        validator()
+                .addInputLines(
+                        "Test.java", "public class Test {", "  public void badMethod(Integer a, Integer b) {}", "}")
```

Comment:
can put a `//` after `"Test.java",` to make the formatter align these one per line